woocommerce/woocommerce-ios
# Blaze Optimization: Remove banner on product list screen

Closes: #10954


## Description


This PR removes the Blaze banner being displayed on the Products list screen.

## Testing instructions


Start a fresh build and go to Products list screen, ensure no Blaze banner is shown on top. A quick read through the codebase should suffice, too.
